### PR TITLE
switch from filesystem to webspace directives

### PR DESCRIPTION
**Pull request by twolfart**
see http://httpd.apache.org/docs/current/sections.html#whichwhen

### CODE REVIEW

File: conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.partials.hbs [hunk 1]
```diff
@@ -6,12 +6,6 @@
 {{/block}}
 
 
-{{~#block "documentRoot"}}
```
Comment (stefanseifert) [inline conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.partials.hbs]: why did you remove the DocumentRoot statement in the PR?

File: conga-aem-definitions/src/main/templates/aem-dispatcher/publish/vhost_publish_tenant.partials.hbs [hunk 2]
```diff
@@ -15,12 +15,6 @@
 {{/block}}
 
 
-{{~#block "documentRoot"}}
```
Comment (stefanseifert) [inline conga-aem-definitions/src/main/templates/aem-dispatcher/publish/vhost_publish_tenant.partials.hbs]: why did you remove the DocumentRoot statement in the PR?

File: conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.partials.hbs (hunk 1 above)
Comment (twolfart) [inline conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.partials.hbs]: DocumentRoot is not required, because disp_apache2 is mapped via location directive to / and handling all the requests.

**Comment by stefanseifert** [comment]
> DocumentRoot is not required, because disp_apache2 is mapped via location directive to / and handling all the requests.

ah, ok - a separate PR or a comment about this had made this more clear

**Comment by twolfart** [comment]
The changes are in the same PR, because they depend on each other: documentroot becomes obsolete with the switch to location.
Thanks for the merge!
